src/main/scala/com/typesafe/sbt/packager/windows/WindowsPlugin.scala

```diff
@@ -96,27 +96,45 @@ object WindowsPlugin extends AutoPlugin {
         val wixConfigFile = (target in Windows).value / ((name in Windows).value + ".wxs")
         IO.write(wixConfigFile, config.toString)
         wixConfigFile
-      }
+      },
+      wixFiles := Seq(wixFile.value)
     ) ++ inConfig(Windows)(Seq(packageBin := {
-      val wixFileValue = wixFile.value
+      val wsxSources = wixFiles.value
       val msi = target.value / (name.value + ".msi")
-      // First we have to move everything (including the wix file) to our target directory.
-      val wix = target.value / (name.value + ".wix")
-      if (wixFileValue.getAbsolutePath != wix.getAbsolutePath) IO.copyFile(wixFileValue, wix)
+
+      // First we have to move everything (including the WIX scripts)
+      // to our target directory.
+      val targetFlat: Path.FileMap = Path.flat(target.value)
+      val wsxFiles = wsxSources.map(targetFlat(_).get)
+      val wsxCopyPairs = wsxSources.zip(wsxFiles).filter {
+        case (src, dest) => src.getAbsolutePath != dest.getAbsolutePath
```

Comment:
This filter would be that `wsxSources` may reference things from `target` ?

Reply:
Yes. The same check is present in the existing code, and I assumed that someone might be relying on that behavior.